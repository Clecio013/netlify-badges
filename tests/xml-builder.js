const getProjectXML = require('../lib/get-project-xml')

test('getProjectXML', () => {
	expect(getProjectXML()).toMatchSnapshot()
	expect(getProjectXML({key: 'value'})).toMatchSnapshot()
	expect(getProjectXML({key: 'value "with quotes"'})).toMatchSnapshot()
})
