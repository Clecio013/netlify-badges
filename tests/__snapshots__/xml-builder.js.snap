// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getProjectXML 1`] = `
<Projects>
<Project />
</Projects>
`;

exports[`getProjectXML 2`] = `
<Projects>
<Project key="value"/>
</Projects>
`;

exports[`getProjectXML 3`] = `
<Projects>
<Project key="value &quot;with quotes&quot;"/>
</Projects>
`;
